Description: Use 4k blocksize in all ext4 mke2fs.conf such that lvm migration
  between non-4k PVs and 4k PVs works irrespective of the volume size.
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1817097


--- e2fsprogs-1.45.1.orig/misc/mke2fs.conf.in
+++ e2fsprogs-1.45.1/misc/mke2fs.conf.in
@@ -14,11 +14,9 @@
 		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
 	}
 	small = {
-		blocksize = 1024
 		inode_ratio = 4096
 	}
 	floppy = {
-		blocksize = 1024
 		inode_ratio = 8192
 	}
 	big = {
